specification.md

```diff
@@ -3915,4 +3915,4 @@ You can find a list of other languages, technologies and specifications related
 ## License
 
 Serverless Workflow specification operates under the
-[Apache License version 2.0](LICENSE).
\ No newline at end of file
+[Apache License version 2.0](LICENSE).
```

Comment:
Wonder why both of your PRs include this change? Maybe some newline char issue?

Reply:
Yeah, it's just an additional line break vim added automatically to the end of the file.

Using `od` we can see the master branch has no line break:

```
╰─ od -c specification.md
# output omitted
0373300       W   o   r   k   f   l   o   w       s   p   e   c   i   f
0373320   i   c   a   t   i   o   n       o   p   e   r   a   t   e   s
0373340       u   n   d   e   r       t   h   e  \n   [   A   p   a   c
0373360   h   e       L   i   c   e   n   s   e       v   e   r   s   i
0373400   o   n       2   .   0   ]   (   L   I   C   E   N   S   E   )
0373420   .
0373421
```

From this branch:

```
╰─ od -c specification.md
# output omitted
0373340   m   d   )   .  \n  \n   #   #       L   i   c   e   n   s   e
0373360  \n  \n   S   e   r   v   e   r   l   e   s   s       W   o   r
0373400   k   f   l   o   w       s   p   e   c   i   f   i   c   a   t
0373420   i   o   n       o   p   e   r   a   t   e   s       u   n   d
0373440   e   r       t   h   e  \n   [   A   p   a   c   h   e       L
0373460   i   c   e   n   s   e       v   e   r   s   i   o   n       2
0373500   .   0   ]   (   L   I   C   E   N   S   E   )   .  \n
0373516
```